package.json

```diff
@@ -156,7 +156,7 @@
   },
   "storybook-deployer": {
     "gitUsername": "GitHub Pages Bot",
-    "gitEmail": "friends@segment.com",
+    "gitEmail": "friends@chord.co",
```

Comment:
I don't think this address exists, although we don't use `storybook-deployer` either, so maybe it doesn't matter.

Reply:
was on the fence about just removing this section all together - think i'll do that, as there are default values that it will fall back to if we ever decide we do want to use it.